apps/nextjs-website/src/_contents/makeDocs.ts

```diff
@@ -52,6 +52,7 @@ export const makeTutorials = ({
           spaceId: dirName,
         },
         bannerLinks: bannerLinks,
+        relatedLinks: [],
```

Comment:
relatedLinks shouldn't be an array, but an object with `title` and `links` attributes:
take a look at `apps/nextjs-website/src/lib/strapi/codecs/RelatedLinksCodec.ts` and `apps/strapi-cms/src/components/common/related-links.json`

Reply:
Shouldn't it be an array of those objects?